<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contacts Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelType">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditType"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelNumber">
        <property name="text">
         <string>Number:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditNumber"/>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="text">
           <string>Add Contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="text">
           <string>Edit Contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="text">
           <string>Delete Contact</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>contacts:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetContacts">
        <property name="toolTip">
         <string>List of Contacts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>favorites:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetFavorites">
        <property name="toolTip">
         <string>Favorites</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayoutSearch"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="favButtonLayout">
      <item>
       <widget class="QPushButton" name="btnAddFav">
        <property name="text">
         <string>Add to Favorites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveFav">
        <property name="text">
         <string>Remove from Favorites</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="saveLoadLayout">
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(65, 74, 255);

color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Save Contacts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(37, 74, 9);

color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Load Contacts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(221, 84, 29);

color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Import Contacts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 0, 0);

color:rgb(0, 0, 0)</string>
        </property>
        <property name="text">
         <string>Export Contacts</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnRefresh">
      <property name="text">
       <string>Refresh Contacts</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
